registry/extension/oras/oras.go

```diff
@@ -21,7 +21,7 @@ const (
 	extensionName          = "artifacts"
 	referrersComponentName = "referrers"
 	namespaceUrl           = "https://github.com/oras-project/artifacts-spec/blob/main/manifest-referrers-api.md"
-	namespaceDescription   = "oras extension enables listing of all reference artifacts associated with subject"
+	namespaceDescription   = "oras extension enables listing of all reference artifacts associated with subject."
```

Comment:
Just curious, when should we put a `.` at the end of the description? I found many descriptions above don't have a `.` at the end.

Reply:
Also, when should we use the term `reference artifacts` and when should we use `referrers`? 😂

Reply:
For the the `.` issue, the namespace level description (which is the line you highlighted) needs to have the period at the end. This is because of how the Discover response's description is formed. It appends the description of the namespace first as it's building the description text for all the endpoints.

Great question about the reference artifacts. This is my bad. I'll switch it.

Reply:
@akashsinghal Thanks for the clarification!